rllib/env/single_agent_env_runner.py

```diff
@@ -197,19 +197,14 @@ def sample(
                     explore=explore,
                     random_actions=random_actions,
                 )
-            # For complete episodes mode, sample as long as the number of timesteps
-            # done is smaller than the `train_batch_size`.
+            # For complete episodes mode, sample a single episode and
+            # leave coordination of sampling to `synchronous_parallel_sample`.
```

Comment:
I very much like this!

Can we add a small TODO comment here that this logic, currently handled by `synchronous_parallel_sample` will eventually be moved fully into `EnvRunnerGroup`? So from the algo, you would do:
```
if self.config.batch_mode == "complete_episodes"
    self.env_runner_group.sample(num_timesteps=[batch size], complete_episodes=True)
```

something like this ^. Don't have to do this in this PR!

Reply:
Awesome. I would love this move!